step-ca.yaml

```
@@ -27,13 +27,16 @@ pipeline:
         github.com/go-jose/go-jose/v3@v3.0.4
         golang.org/x/oauth2@v0.27.0
         golang.org/x/crypto@v0.35.0
+        golang.org/x/net@v0.36.0
 
-  - runs: |
-      CGO_ENABLED=0 go build -v \
-        -ldflags='-w -X "main.Version=${{package.version}}"' \
-        -o "${{targets.destdir}}/usr/bin/step-ca" ./cmd/step-ca
-
-  - uses: strip
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/step-ca
+      output: step-ca
+      ldflags: |
+        -X main.Version=${{package.version}}
+        -X main.BuildTime=$(date -u '+%Y-%m-%d %H:%M UTC')
```

```suggestion
        -X main.BuildTime=$(TZ=UTC date -u +"%Y-%m-%dT%H:%M:%SZ")
```
UTC and matches what upstream is using.

---

`-u` already forces time zone to UTC so setting `TZ=UTC` would be redundant in this case.

---

Yes that is true :)